src/Microsoft.Dnx.Tooling/Restore/RestoreOperations.cs

```diff
@@ -26,6 +26,45 @@ public RestoreOperations(IReport report)
             var sw = new Stopwatch();
             sw.Start();
 
+            var dependencies = new List<LibraryDependency>();
+            if (context.RuntimeSpecs != null && !string.IsNullOrEmpty(context.RuntimeName))
+            {
+                // HACK(davidfowl): This is making runtime.json support package redirects
+
+                // Look up any additional dependencies for this package
+                var runtimeDependencies = context.RuntimeSpecs
```

Comment:
Ah, the LINQ. It's the equivalent of https://github.com/NuGet/NuGet3/blob/dev/src/NuGet.RuntimeModel/RuntimeGraph.cs#L147 (which is what the NuGet3 code called).